### ISSUE TITLE
Don't require private repo access for the GitHub app

### ISSUE THREAD

**Issue by epicfaace** [suttacentral/bilara-data#550]
I'd love to try out Bilara with my GitHub account, but I don't want to give it access to all my private repository data. Is this permission necessary -- would it be possible to remove this permission ask so that Bilara doesn't need access to all a user's private repository data?

![image](https://user-images.githubusercontent.com/1689183/112699555-783af100-8e62-11eb-816c-c735373c397e.png)

**Comment by sujato** [comment]
Hi, I'm not sure what you mean by trying out Bilara: do you mean to fork it and run your own instance?

As for your request, the simple answer is, I don't know. But Bilara doesn't know anything about what's going on in your repo, nor does it care. It only does what it says on the can, and has no capacity for sniffing out data or compromising security, if that's your concern.

**Comment by epicfaace** [comment]
Hi -- what I meant is, I went to https://bilara.suttacentral.net/ and clicked "Login" to log in with Github.



> But Bilara doesn't know anything about what's going on in your repo, nor does it care. It only does what it says on the can, and has no capacity for sniffing out data or compromising security, if that's your concern.

That's great, that's what I surmised. In that case, it might be possible to configure the GitHub login integration so that it doesn't ask you for _all_ public / private repositories and instead asks you for a more limited set of scopes. That's essentially the suggestion I was making in this issue.
